seatunnel-core/seatunnel-core-starter/pom.xml

```diff
@@ -38,6 +38,12 @@
             <version>${project.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.seatunnel</groupId>
+            <artifactId>seatunnel-engine-common</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
```

Comment:
-1. we should not include this dependency. The starter for all engine, not only for zeta.

Reply:
get